apps/judicial-system/web/src/routes/Prosecutor/Indictments/Processing/Processing.tsx

```diff
@@ -96,6 +101,36 @@ const Processing: FC = () => {
     [updateDefendantState, setWorkingCase, workingCase.id, updateDefendant],
   )
 
+  const handleHasCivilClaimsChange = async (hasCivilClaims: boolean) => {
+    setHasCivilClaimsChoice(hasCivilClaims)
+
+    setAndSendCaseToServer(
+      [{ hasCivilClaims, force: true }],
+      workingCase,
+      setWorkingCase,
+    )
+
+    if (hasCivilClaims === false) {
```

Comment:
I wonder if removing the civil demands text and civil claim files is perhaps a bit hostile to the user. If the user subsequently changes this to true, then the civil demands need to be typed in again and the civil claim files uploaded again.

I suggest we leave this as is, but discuss with the team.